tests/workflows/multiple_node_adornments/tests/test_workflow.py

```diff
@@ -0,0 +1,20 @@
+from vellum.workflows.workflows.event_filters import all_workflow_event_filter
+
+from tests.workflows.multiple_node_adornments.workflow import MultipleNodeAdornmentsExample, StartNode
+
+
+def test_stream_workflow__happy_path():
```

Comment:
this test was already passing, but I had already wrote it, and I imagine will be useful to keep for this intricate use case